db/migrate/20150419001730_create_pieces.rb

```diff
@@ -2,6 +2,7 @@ class CreatePieces < ActiveRecord::Migration
   def change
     create_table :pieces do |t|
       t.integer :game_id
+      t.integer :user_id
```

Comment:
You shouldn't go back and modify an old migration file. You should always create a new migration to add the column.